packages/service/tests/unit/test_service.py

```diff
@@ -446,11 +446,22 @@ def test___service_config___create_measurement_service_with_version___version_di
     assert "Version mismatch" in str(version_error.value)
 
 
+def test___service_config___create_measurement_service_with_version___service_config_has_no_version(
+    test_assets_directory: pathlib.Path,
+):
+    measurement_service = MeasurementService(
+        service_config_path=test_assets_directory / "example.NoVersion.serviceconfig",
+        version="2.0.1",
+    )
+
```

Comment:
These tests are generating warnings: 

```
============================== warnings summary ===============================
tests/unit/test_service.py::test___service_config___create_measurement_service_with_version___version_differs_from_service_config
tests/unit/test_service.py::test___service_config___create_measurement_service_with_version___service_config_has_no_version
tests/unit/test_service.py::test___service_config___create_measurement_service_with_version___version_is_used[example.serviceconfig-1.0.1]
tests/unit/test_service.py::test___service_config___create_measurement_service_with_version___version_is_used[example.v1.serviceconfig-1.0.1]
  D:\a\measurement-plugin-python\measurement-plugin-python\packages\service\ni_measurement_plugin_sdk_service\measurement\service.py:222: DeprecationWarning: The 'version' constructor parameter is deprecated. Specify the version using the 'version' field in the .serviceconfig file instead.
    warnings.warn(

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
- generated xml file: D:\a\measurement-plugin-python\measurement-plugin-python\packages\service\test_results\nims-windows-latest-py3.9-all-extras.xml -
```

Fixed in https://github.com/ni/measurement-plugin-python/pull/942